app/frontend/src/components/form/Step3.vue

```diff
@@ -1,163 +1,202 @@
 <template>
   <v-container>
     <div class="hide-on-review">
-      <h2 class="pb-8">Before you begin planting, please certify</h2>
+      <h2 class="pb-8">Before you begin planting, please check all that apply</h2>
       <hr class="orange" />
     </div>
 
+    <BaseInfoCard class="hide-on-review mt-5 mb-8">
+      Please note questions
+      <strong>2, 3, and 4</strong> are not relevant if your accommodation is a private residence.
```

Comment:
minor point: this could be 16px font as well.

Reply:
Thanks! that's supposed to be 16. Fixing.